pest/src/parser_state.rs

```diff
@@ -86,6 +87,26 @@ pub enum MatchDir {
     TopToBottom,
 }
 
+/// A literal value, for use in `push_literal`. Use the `Static` variant when possible (for example, within a derived
```

Comment:
coderabbitai wants all of this new code guarded by `#[cfg(feature = "grammar-extras")]` -- what do you think?

I don't think there's any harm in doing so, but I also don't know if there's a need. My understanding (please correct me if I'm wrong!) is that the feature flag is there so that users can opt into new grammar features, in case those collide with their existing grammars. For example, someone could have created a grammar with a rule named `PUSH_LITERAL`. But that concern doesn't apply to this file.

Reply:
Ah, okay -- good to know. I think by that reasoning, I still don't need to use `#[cfg` for this, right?